Sources/TuistDependencies/SwiftPackageManager/Utils/PackageInfoMapper.swift

```diff
@@ -106,6 +114,8 @@ public protocol PackageInfoMapping {
 }
 
 public final class PackageInfoMapper: PackageInfoMapping {
+    fileprivate static let predefinedSourceDirectories = ["Sources", "Source", "src", "srcs"]
```

Comment:
I'd be curious to know if SPM has some precedence on the individual names

Reply:
Theyre found here (: https://github.com/apple/swift-package-manager/blob/751f0b2a00276be2c21c074f4b21d952eaabb93b/Sources/PackageLoading/PackageBuilder.swift#L488